typescript/sdk/src/deploy/middleware/deploy.ts

```diff
@@ -31,6 +31,7 @@ export class InterchainAccountDeployer<
     multiProvider: MultiProvider<Chain>,
     configMap: ChainMap<Chain, InterchainAccountConfig>,
     protected core: HyperlaneCore<Chain>,
+    protected create2salt = 'asdasdsd',
```

Comment:
intentional? if we need a random value, how about `hyperlane`?

Reply:
indeed just a random value, and yeah we should change that for the next time. Right now that is the salt for the deployed contracts, so don't want to change that, otherwise would need to redeploy

Reply:
Will add a TODO here